doc/Relationships.md

```diff
@@ -223,6 +223,11 @@ When displaying a post:
 
 ![embedded_list in editionView](images/embedded_list_in_editionView.png)
 
+Caution: If you want to use [`Field.template()`](./Theming.md#customizing-the-template-for-a-given-field)
```

Comment:
Right: we should think to fix this black magic. :+1: 

Can you add your explanation directly into the README?